tests/test_color.py

```diff
@@ -40,6 +40,12 @@
         ('rgba(00,0,128,0.6  )', (0, 0, 128, 0.6)),
         ('rgba(0, 0, 128, 0)', (0, 0, 128, 0)),
         ('rgba(0, 0, 128, 1)', (0, 0, 128)),
+        ('rgb(0 0.2 205)', (0, 0, 205)),
+        ('rgb(0 0.2 205 / 0.6)', (0, 0, 205, 0.6)),
+        ('rgb(0 0.2 205 / 60%)', (0, 0, 205, 0.6)),
+        ('rgba(0 0 128)', (0, 0, 128)),
+        ('rgba(0 0 128 / 0.6)', (0, 0, 128, 0.6)),
+        ('rgba(0 0 128 / 60%)', (0, 0, 128, 0.6)),
```

Comment:
I think we should add some hypothesis tests for color parsing, perhaps using another library like [`webcolors`](https://pypi.org/project/webcolors/) to compare against.